pkg/providers/openshift/provider_postgres.go

```diff
@@ -443,7 +444,7 @@ func buildDefaultPostgresPodContainers(ps *v1alpha1.Postgres) []v1.Container {
 	return []v1.Container{
 		{
 			Name:  ps.Name,
-			Image: "registry.redhat.io/rhscl/postgresql-96-rhel7",
+			Image: "registry.redhat.io/rhscl/postgresql-10-rhel7",
```

Comment:
Why are we changing this image?

Reply:
Looks like a cherry pick from my old branch which should be merged now, just needs a rebase I think

Reply:
Why was the image changed, have the products changed to this version?

Reply:
This is to fix the bug with 3scale where the system-app pods kept crashing. 3scale 2.6+ requires postgres 10+

Reply:
Perfect, thanks dimitra.